user/useradm.go

```diff
@@ -30,19 +34,19 @@ var (
 	ErrUserNotInitial = errors.New("user database not empty")
 )
 
-type UserAdmApp interface {
+type App interface {
 	// Login accepts email/password, returns JWT
 	Login(email, pass string) (*jwt.Token, error)
-	CreateUser(u *UserModel) error
-	CreateUserInitial(u *UserModel) error
+	CreateUser(u *model.User) error
+	CreateUserInitial(u *model.User) error
 	Verify(token *jwt.Token) error
 
 	// SignToken returns a function that can be used for generating a signed
 	// token using configuration & method set up in UserAdmApp
```

Comment:
It's already included in https://github.com/mendersoftware/useradm/pull/44/commits/124b49f64fab72f56168831813e8f4f8747c2df0